### ISSUE TITLE
SCI KIT UFMPACK not working in python 3.6.6

### ISSUE THREAD

**Issue by monikrish2605** [scikit-umfpack/scikit-umfpack#58]

**Comment by rgommers** [comment]
it works for other people, please provide more details like how you installed it and what error you are seeing

**Comment by monikrish2605** [comment]
Hi rgommers,
Thanks for reply.
 
I installed using 
PIP install scikit-ufmpack
it got installed properly.but later when tried for checking some example code, it is saying
 **import scikits.ufmpack
Traceback (most recent call last):

  File "<ipython-input-7-77994a876d93>", line 1, in <module>
    import scikits.ufmpack

ModuleNotFoundError: No module named 'scikits.ufmpack'**

I was having Python 3.7 , i want to install sfepy as i said above , so when i was in the step of installing,clinicalgraphics, wxpython and mayavi through  Anaconda command 
"conda install -c anaconda mayavi"
my python got automatically downgraded to 3.6.6
Also i attempted to install scikit-ufmpack also through anaconda command but it didnt work so i tried with "PIP install scikit-ufmpack",it installed but giving above error

**Comment by rc** [comment]
Hi @monikrish2605, did you install umfpack with pip inside your conda environment?

Also note, that scikit-umfpack is only an optional dependency of sfepy - you do not need it to run it. Anyway, this is not a place to discuss sfepy installation.

**Comment by monikrish2605** [comment]
yes i installed using PIP ..because "conda install -c conda-forge scikit-umfpack" this was available for only linux and mac OS

**Comment by rgommers** [comment]
If you're on Windows, you really want a wheel for scikit-umfpack. The 0.3.2 release is source-only, hence it's trying to build from source (needs a Fortran-compiler etc., don't go there). For Python 3.6 there is a wheel available for 0.3.1, so do:

    pip install scikit-umfpack=='0.3.1'

inside your conda environment.

And as @rc says, you don't really need it for SfePy so if my suggestion above still doesn't work for you then best to just skip installing this package.

**Comment by monikrish2605** [comment]
Thanks @rc and @rgommers  , i will try with .whl which you mentioned above.if  that doesnot work , i will skip that as you said, but if i skip it ,what i should do when i encountered to use  solver option in that module , is there any alternate for that ?

**Comment by monikrish2605** [comment]
Sorry i am new to python env(for using FEM), hence please bare with me if my question is apparently naive

**Comment by rgommers** [comment]
> what i should do when i encountered to use solver option in that module , is there any alternate for that ?

The SciPy solver that uses scikit-umfpack under the hood will still work, just slower likely. So you should not notice. If you do get an exception, it's likely a SfePy bug.

**Comment by monikrish2605** [comment]
Thanks rgommers

**Comment by rc** [comment]
FYI: thanks to @matthew-brett's help, the 0.3.2 wheels are now available for linux and os x.

**Comment by rc** [comment]
I am not very knowledgeable in CI/build issues, especially on windows, so any help would be appreciated, see https://github.com/scikit-umfpack/scikit-umfpack-wheels/issues/2 for a background.
